src/analyses/ai.h

```diff
@@ -11,10 +11,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <map>
 #include <iosfwd>
-#include <sstream>
 
 #include <util/json.h>
 #include <util/xml.h>
+#include <util/expr.h>
+#include <util/simplify_expr.h>
```

Comment:
Good point; have moved the simplify_expr.h into ai.cpp.  I'm mildly reluctant to forwards declare instead of include except where necessary.